src/watchmaker/workers/salt.py

```diff
@@ -834,6 +896,7 @@ def install(self):
         self.log.info('Refreshing package database...')
         self.run_salt('pkg.refresh_db')
 
+        self._install_pip_modules()
```

Comment:
would prefer to do this in the same place on windows as linux, just after `self._install_package()`, unless there is a good reason not to.